evm_loader/program/src/types/transaction.rs

```diff
@@ -266,13 +266,123 @@ impl rlp::Decodable for AccessListTx {
     }
 }
 
```

Comment:
We don't do it elsewhere (yet) and it may be a useless idea, but thinking in terms of optimizing space - should we consider reordering fields in favour of optimising for struct packing here? Like I said - it might be useless trying to save a few bytes, but just thought I would bring it up (at least to keep in the back of our collective minds for the future).